### PR TITLE
Higher auto refinement

### PR DESCRIPTION
**Pull request by najlkin**
Rebalanced refinements to match the original ones after the new auto refinement in GLVis/glvis#285

### CODE REVIEW

File: streams/ex1.saved [hunk 1]
```diff
@@ -98529,4 +98529,4 @@ Ordering: 0
 0.015023882
 0.0068553618
 keys
-mAXXXXXXXXXXXXXXXXXXZi66666666666666666666666666666666666666666666666i
+OOOmAXXXXXXXXXXXXXXXXXXZi66666666666666666666666666666666666666666666666i
```
Comment (tzanio) [inline streams/ex1.saved:98532]: I am confused by `OOO` here, does this do anything?

Comment (najlkin) [reply streams/ex1.saved:98532]: Yes, it reduces refinement. Unfortunately, `O` in 2D means switching the function of `o`, but in 3D it is decreasing the refinement directly. This is not confusing at all to the users 🙃

Comment (tzanio) [reply streams/ex1.saved:98532]: Oh, right 🤦

**Comment by justinlaughlin** [comment]
Ooooo (looks good)

**Comment by najlkin** [comment]
We need some Ooooos in GLVis/glvis#285 mainly 😄

**Review by tzanio — APPROVED** [review]

**Pull request merged**